Tests/test_Phylo.py

```diff
@@ -497,6 +497,15 @@ def test_split(self):
             self.assertEqual(clade.name, name)
             self.assertEqual(clade.branch_length, blen)
 
+    def test_resolve_polytomies(self):
+        """TreeMixin: resolve_polytomies() method."""
+        tree = list(Phylo.parse(EX_NEXUS, "nexus"))[2]
+        # tree1 = (1,(2,(4,(5,6,(7,(8,(9,3)))))));
+        #                   ^ ^ ^ multifurcation
+        self.assertFalse(tree.is_bifurcating())
+        tree.resolve_polytomies(shuffle=False, recursive=True)
+        self.assertTrue(tree.is_bifurcating())
+
```

Comment:
Maybe add a check on the string representation of the tree before/afterwards too?

Reply:
Thanks for the suggestion, I've modified the test to compare with an expected resolved tree.